concent_api/command_line_tool/message_extractor.py

```diff
@@ -0,0 +1,122 @@
+from base64 import b64decode
+import importlib
+import re
+import random
+import string
+from typing import Dict, Text, Any, List, Union
+from golem_messages.datastructures import FrozenDict
+from golem_messages.message import Message
+
+JsonType = Dict[Text, Any]
+Task = FrozenDict
+
+
+def make_random_string(length=None, chars=None):
+    length = length if length is not None else 36
+    chars = chars if chars is not None else (string.ascii_letters + string.digits)
+
+    return ''.join(random.choice(chars) for _ in range(length))
+
+
+def split_uppercase(message):
+    return (''.join([(word[:1].lower() + word[1:]) for word in re.sub(r'([A-Z])', r'_\1', message)]))[1:]
+
+
+def find_modules():
+    message_tasks = importlib.import_module("golem_messages.message.tasks")
```

Comment:
Why is this not imported directly at the beginning of the file ? I guess the requirement is to run those tests on Concent virtual env, which contains golem_messages.

Reply:
Done.